proto/cosmos/nft/v1beta1/nft.proto

```diff
@@ -43,6 +43,9 @@ message NFT {
   // uri_hash is a hash of the document pointed by uri
   string uri_hash = 4;
 
+  // send_enabled is a flag that enables MsgSend
```

Comment:
Do we need a migration for setting send enabled to false to all NFTs?

Reply:
You're right. We need it.
Let me fix all things you reviewed.